cart/src/components/Cart.js

```diff
@@ -0,0 +1,38 @@
+import "../styles/Cart.css";
+
+const Cart = ({ cartArray, totalPrice, handleIncDec, removeCart }) => {
+  const cards =
+    cartArray.length === 0 ? (
+      <h1 className="empty">Cart is Empty</h1>
+    ) : (
+      cartArray.map((items) => (
+        <div className="cart-items" key={items.id}>
+          <img src={items.logoURL} alt="cart" />
```

Comment:
pls add a dynamic value for the `alt` tag here. It should not be same for all images